### ISSUE TITLE
Exception caught during install: Error processing 'astra_camera'

### ISSUE THREAD

**Issue by dimadobriy** [DFKI-NI/mobipick#2]
Hi there!
Thanks for sharing your project! It looks amazing!
Hope, you can help to overcome the following error at the installation stage:

running script ```install-deps.sh``` I've got the following error:

```Exception caught during install: Error processing 'astra_camera' : [astra_camera] Checkout of https://github.com/orbbec/ros_astra_camera.git version master into /misis/mir_ws/src/astra_camera failed.```
```ERROR in config: Error processing 'astra_camera' : [astra_camera] Checkout of https://github.com/orbbec/ros_astra_camera.git version master into /misis/mir_ws/src/astra_camera failed.```

I use Ubuntu 20.04, ROS-noetic.

Thanks for your help.

Best regards,
Dmitrii

**Comment by mintar** [comment]
That sounds like a network error when trying to reach github. Just try again.

**Comment by dimadobriy** [comment]
Thanks for your fast response.
The network was OK, I'm sure. I've downloaded it manually.
The environment was built without any errors. 

Also, I noticed that additional libraries and modules are required to build the project (not mentioned in the guidelines and dependencies).
I packed it all in a Docker-container. If you're interested, I can make a pull request. Just let me know :)

Regards,
Dmitrii
